APIWrapper/SchoolScheduleProxy.cs

```diff
@@ -38,7 +38,22 @@ public ScheduleWeekContent GetBySubjectCode(string subjectCode)
 
         private ScheduleWeekContent CallScheduleContentApi(int type, string requestContent)
         {
-            var address = $"{URL}/{SCHEDULE_CONTENT_URL}?m={type}&id={Uri.EscapeUriString(requestContent)}";
+            DateTime localDate = DateTime.Now;
+            DateTime winterSemesterStart = new DateTime(localDate.Year, 9, 1); //start of winter semester 1.9.
+            DateTime winterSemesterEnd = new DateTime(localDate.Year, 3, 1); //end of winter semester and start of summer semester 1.3.
+            DateTime summerSemesterEnd = new DateTime(localDate.Year, 7, 1); //end of summer semester 1.7.
+            var address = $"{URL}/{SCHEDULE_CONTENT_URL}?m={type}&id={Uri.EscapeUriString(requestContent)}&r={localDate.Year}";
```

Comment:
What about to put this constat at least to some const or better to configuration?
And all this new part as separate method, which will return the result L or Z. And may be also to separate class

Reply:
It is solved?